sansio_lsp_client/client.py

```diff
@@ -120,7 +271,7 @@ def _handle_response(self, response: Response) -> Event:
 
         if request.method == "initialize":
             assert self._state == ClientState.WAITING_FOR_INITIALIZED
-            self._send_notification("initialized")
+            self._send_notification("initialized", params={}) # 'gopls' doesnt recognise 'None' 'params'
```

Comment:
What does the comment mean? I don't understand it.

Reply:
golang server - 'gopls' - does not recognize 'initialized' notification if `params` is `None`

Reply:
```suggestion
            self._send_notification("initialized", params={}) # params=None doesn't work with gopls
```